crowdfunding/projects/serializers.py

```diff
@@ -0,0 +1,105 @@
+from django.contrib.auth.models import User
+from .models import Project, Pledge, Comment, Category, Favourite
+from rest_framework import serializers
+from users.serializers import CustomUserSerializer
+
+'''Serializer / Forms'''
+
+
+class CommentSerializer(serializers.ModelSerializer):
+    commentator = serializers.ReadOnlyField(source='commentator.username')
+    # project = serializers.SlugRelatedField(
+    #     queryset=Project.objects.all(), slug_field="title")
+
+    class Meta:
+        model = Comment
+        fields = '__all__'
+        # fields = ['id', 'project', 'title', 'content', 'author']
+        # read_only_fields = ['id']
+
+    def create(self, validated_data):
+        return Comment.objects.create(**validated_data)
+
+    def update(self, instance, validated_data):
+        instance.save()
+        return instance
```

Comment:
i don't think you need to have create or update as the model serializer does this for you automatically